app/models/host/managed.rb

```diff
@@ -848,6 +848,18 @@ def setup_clone
     @old = super { |clone| clone.interfaces = self.interfaces.map {|i| setup_object_clone(i) } }
   end
 
+  # rebuilds orchestration configuration for a host
+  # checks all Orchestration:: modules mixed into provisioning interface and calls rebuild_* if such method is defined
+  # returns  : Hash with 'true' if rebuild was a success for a given key (Example: {:tftp => true, :dhcp => false})
+  def recreate_config
+    result = {}
+    nic = interfaces.select(&:provision).first
```

Comment:
You'd probably want to do *all* NICs, because otherwise things such as DHCP configs on secondary NICs won't get recreated.